Gemfile

```diff
@@ -203,3 +203,17 @@ end
 GemfileHelper.parse_each_agent_gem(ENV['ADDITIONAL_GEMS']) do |args|
   gem *args
 end
+
+if ENV['HUGINN_AGENT_SPEC_RUNNER']
+  base_path = File.expand_path('../../../', __FILE__)
```

Comment:
Actually, even better if it can be just in the spec runner. See https://github.com/cantino/huginn_agent/pull/15#pullrequestreview-27722414